### ISSUE TITLE
Unexpected results

### ISSUE THREAD

**Issue by Scotchester** [mislav/gh-branch#2]
Congrats on being featured in the GitHub CLI 2.0 release blog :)

I'm seeing some unexpected results when I run the extension:

![gh branch output showing four branches: gh-pages, develop, production, and staging](https://user-images.githubusercontent.com/1044670/130667833-569a015e-5ad7-480e-b90d-4f6111962cdc.png)

I have two other local feature branches (one whose PR is merged, one whose PR is not) that are not showing up.

Not sure if this is relevant or not, but this repo's origin remote is on a GitHub Enterprise Server instance, but the GitHub CLI is authenticated to it.

Thanks in advance for any help you can offer!

**Comment by mislav** [comment]
Thanks for reporting! Because I don't know how to handle this situation better, I simply ignore the error when gh was unable to fetch information from the API. To get PR information working, you will have to authenticate to your GHES instance!

**Comment by Scotchester** [comment]
Ah, but I _am_ authenticated to my GHES instance :)

I am _also_ authenticated to GH.com. Is it possibly getting confused by being authenticated to both?

```bash
cranfill@MT-308679 WCP % gh auth status
github.com
  ✓ Logged in to github.com as Scotchester (/Users/cranfill/.config/gh/hosts.yml)
  ✓ Git operations for github.com configured to use ssh protocol.
  ✓ Token: *******************

<redacted ghe hostname>
  ✓ Logged in to <redacted ghe hostname> as cranfill (/Users/cranfill/.config/gh/hosts.yml)
  ✓ Git operations for <redacted ghe hostname> configured to use ssh protocol.
  ✓ Token: *******************
```

**Comment by Scotchester** [comment]
Good news, though, the fix you put in for #4 fixed my missing branches! They all used `type/` branch type prefixes.
